apps/web/components/walkthrough/Walkthrough.tsx

```diff
@@ -59,10 +58,8 @@ const Walkthrough = observer((): JSX.Element => {
           key={`walkthrough-wrapper-${currentItemId}`}
         >
           {currentResult ? (
-            <Result
-              displayMessage={currentResult.resultDisplayMessage}
-              relatedWalkthroughs={relatedWalkthroughs}
-            />
+            // TODO - HOUSNAV-191
```

Comment:
nothing is commented out here, is there still work to be done? not sure what the TODO is here